### PR TITLE
updated vi-faq.md and vi-github-and-markdown.md (issue #1543)

### PR DESCRIPTION
**Pull request by azywong**
Updated the faq and added a link to it on the relevant part of the github and markdown page. 

https://rawgit.com/azywong/azywong.github.io/master/#!pages/vi/vi-faq.md
https://rawgit.com/azywong/azywong.github.io/master/#!pages/vi/vi-github-and-markdown.md

### CODE REVIEW

File: pages/vi/vi-github-and-markdown.md [hunk 1]
```diff
@@ -58,15 +58,17 @@ First, you will need to find the correct GitHub repository to fork. Forking is e
 
 ### Go to Settings and rename your repository
 
-After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username. See example below.
+After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username.  See example below.
 
 ![Renaming Repository](images/vi-rename-repository.png)
 
+Check [FAQ Q16](http://open-learning-exchange.github.io/#!pages/vi/vi-faq.md) if you already have a repository named YourUserName.github.io.
```
Comment (xyb994) [inline pages/vi/vi-github-and-markdown.md]: - it would be better to use relative links whenever possible, please see #515
- could you add Q16's id `#16...` to the end of the link so it goes straight to Q16?

File: pages/vi/vi-github-and-markdown.md [hunk 2]
```diff
@@ -58,15 +58,17 @@ First, you will need to find the correct GitHub repository to fork. Forking is e
...
+
 ### Post a link to your repository in chat
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-> @dogi  please look at  https://YourUserName.github.io 
+> @dogi  please look at  https://YourUserName.github.io
```
Comment (xyb994) [inline pages/vi/vi-github-and-markdown.md]: I understand trailing whitespace is not optimal, but these change are not related to the original issue and it makes the PR seem to have more lines of change than it needed. If you are actually work on those lines that have trailing whitespace, it would be fine to remove them.

**Comment by ofrank23447** [comment]
lorugrant is correct. Please check http://open-learning-exchange.github.io/#!./pages/vi/vi-github-issues.md under "Create a New Branch" section for this problem.

**Comment by kylemathias** [comment]
If anyone is ever working on the master, feel free to close it out.
